src/jdk.jdi/share/classes/com/sun/jdi/ThreadReference.java

```diff
@@ -375,10 +375,9 @@ List<MonitorInfo> ownedMonitorsAndFrames()
      * All <code>StackFrame</code> objects for this thread are
      * invalidated.
      * <p>
-     * The target VM may not support, or may only provide limited support,
-     * for popping stack frames when the thread is a virtual thread.
-     * It may, for example, only support this operation when the virtual
-     * thread is suspended at a breakpoint or singlestep event.
+     * This method may be used to pop frames of a virtual thread when
+     * it is suspended at an event. An implementation may support popping
+     * the frames of a suspended virtual thread in other cases.
```

Comment:
This text looks okay but I notice a bit further up that it says "The specified thread must be suspend.". I assume that should be "This thread must be suspended."

Reply:
Ok. Fixed.